src/tree/registries/gitLab/GitLabAccountTreeItem.ts

```diff
@@ -21,14 +22,12 @@ export class GitLabAccountTreeItem extends AzExtParentTreeItem implements IRegis
     public label: string = 'GitLab';
     public childTypeLabel: string = 'project';
     public baseUrl: string = 'https://gitlab.com/';
-    public cachedProvider: ICachedRegistryProvider;
 
     private _token?: string;
     private _nextLink?: string;
 
-    public constructor(parent: AzExtParentTreeItem, provider: ICachedRegistryProvider) {
+    public constructor(parent: AzExtParentTreeItem, protected readonly provider: IRegistryProvider, public readonly cachedProvider: ICachedRegistryProvider) {
```

Comment:
This isn't strictly necessary to refactor all the way down, the child nodes don't actually use the `provider`. Could just accept it in the constructor (to fit `IRegistryProvider`) and ignore it.